### PR TITLE
Update lodash imports

### PR DESCRIPTION
**Pull request by vojtechszocs**
Follow-up to #1708, this PR removes the `lodash` vs. `lodash-es` dependency duality.

During webpack build, `lodash` imports get transformed into corresponding `lodash-es` imports.

#### `dependencies`

- added explicit `lodash@4.x` alongside existing `lodash-es@4.x`
- both ^^ now resolve to the same version

```sh
$ cat node_modules/lodash/package.json | jq .version
"4.17.11"
$ cat node_modules/lodash-es/package.json | jq .version
"4.17.11"
```

#### `devDependencies`

- removed `@types/lodash-es`
- added `@types/lodash-es`

The last commit in this PR replaces the remaining `lodash-es` imports with `lodash`.

### CODE REVIEW

File: frontend/packages/console-app/src/__tests__/extension-checks/nav-items.spec.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { testedRegistry } from '../plugin-test-utils';
```
Comment (benjaminapetersen) [inline frontend/packages/console-app/src/__tests__/extension-checks/nav-items.spec.ts]: This PR looks like more than updates to lodash imports?

**Comment by benjaminapetersen** [comment]
Several hundred files, it may be helpful to split into 2 commits, the first for the configuration changes you made, the second for all the files that are updated.  I'm sure the configuration changes are what need the review attention.

File: frontend/packages/console-app/src/__tests__/extension-checks/nav-items.spec.ts (hunk 1 above)
Comment (vojtechszocs) [reply frontend/packages/console-app/src/__tests__/extension-checks/nav-items.spec.ts]: Hi Ben, this PR is a follow-up to un-merged #1708, above change comes from that PR.

**Comment by vojtechszocs** [comment]
> Several hundred files, it may be helpful to split into 2 commits, the first for the configuration changes you made, the second for all the files that are updated. I'm sure the configuration changes are what need the review attention.

Agreed, will do.

**Comment by vojtechszocs** [comment]
The first 2 commits come from different un-merged PRs, please review the last 3 commits only.

**Comment by vojtechszocs** [comment]
Putting this on hold for now, I'll extract the dependency update commit into a separate PR.

/hold

**Pull request closed without merging**